feeds/wifi-ax/hostapd/patches/f00-021-chan-switch-check-dfs-channel-availability.patch

```diff
@@ -0,0 +1,31 @@
+Index: hostapd-2020-07-02-58b384f4/src/ap/ubus.c
+===================================================================
+--- hostapd-2020-07-02-58b384f4.orig/src/ap/ubus.c
++++ hostapd-2020-07-02-58b384f4/src/ap/ubus.c
+@@ -1016,12 +1016,20 @@ hostapd_switch_chan(struct ubus_context
+ 		css.freq_params.center_freq1,
+ 		css.freq_params.center_freq2);
+ 
+-	if ((chan->flag & HOSTAPD_CHAN_RADAR) &&
+-	    ((chan->flag & HOSTAPD_CHAN_DFS_MASK)
+-	     != HOSTAPD_CHAN_DFS_AVAILABLE)) {
+-		wpa_printf(MSG_INFO, "%s: DFS chan need CAC", __func__);
+-		hostapd_switch_chan_dfs(iface, &css);
+-		return UBUS_STATUS_OK;
++	if (chan->flag & HOSTAPD_CHAN_RADAR) {
++		if ((chan->flag & HOSTAPD_CHAN_DFS_MASK) ==
++		     HOSTAPD_CHAN_DFS_UNAVAILABLE) {
++			wpa_printf(MSG_WARNING, "DFS chan %d not Available",
++				   chan->chan);
++			return UBUS_STATUS_NOT_SUPPORTED;
++		}
++		
++		if ((chan->flag & HOSTAPD_CHAN_DFS_MASK)
```

Comment:
Could line 23 "if" be an "else" to line 16 "if"?  More readable, since second if just negates the first,

Reply:
done